src/sdk/config_fs.js

```diff
@@ -1182,15 +1182,31 @@ class ConfigFS {
         }
         const running_code_config_dir_version = this.config_dir_version;
         const system_config_dir_version = system_data.config_directory.config_dir_version;
+        const ver_comparison_err = this.compare_host_and_config_dir_version(running_code_config_dir_version, system_config_dir_version);
+        if (ver_comparison_err !== undefined) {
+            throw new RpcError('CONFIG_DIR_VERSION_MISMATCH', ver_comparison_err);
+        }
+    }
+
+    /**
+     * compare_host_and_config_dir_version compares the version of the config dir in the system.json file 
+     * with the config dir version of the running host
+     * @param {String} running_code_config_dir_version 
+     * @param {String} system_config_dir_version 
+     * @returns {String | Undefined}
+     */
+    compare_host_and_config_dir_version(running_code_config_dir_version, system_config_dir_version) {
         const ver_comparison = version_compare(running_code_config_dir_version, system_config_dir_version);
+        dbg.log0('config_fs.compare_host_and_config_dir_version', running_code_config_dir_version, system_config_dir_version, ver_comparison);
```

Comment:
I think it would be easier to read the logs if we printed the variables names/meanings and not just the values, for example:
```suggestion
        dbg.log0(`config_fs.compare_host_and_config_dir_version: ver_comparison ${ver_comparison} running_code_config_dir_version ${running_code_config_dir_version} system_config_dir_version ${system_config_dir_version}`);
```